commands/env/cancel.go

```diff
@@ -60,7 +60,7 @@ func cancelEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("cancel", "environment", id, "", params), nil)
+	_, err = requester.Do(http.MethodPost, uri.CreateResourceURI("cancel", "environment", id, "", params), nil)
```

Comment:
nit: scoping.  `if _, err := ... ; err != nil {`

Reply:
I avoided that in these requester calls because the lines get too long. But I changed this elsewhere...